03_nyc311/m-nyc311.yml

```diff
@@ -0,0 +1,127 @@
+glean: "1.0"
+type: model
+name: NYC 311
```

Comment:
I made a model in one of the demo projects on stage and then viewed the model DataOps config via the UI. It looks like what you've got would work:

```
glean: '1.0'
type: model
grn: m:UXHmlxgRarJHALmp
name: quickstart.nyc_311 Query
source:
  connectionName: bigquery-template
  sql: |
    select * from quickstart.nyc_311
...
```